samples/emoji-search/presenters/src/jsMain/kotlin/app/cash/zipline/samples/emojisearch/presentersJs.kt

```diff
@@ -16,11 +16,10 @@
 package app.cash.zipline.samples.emojisearch
 
 import app.cash.zipline.Zipline
-import kotlinx.serialization.ExperimentalSerializationApi
 
 private val zipline by lazy { Zipline.get() }
 
-@OptIn(ExperimentalSerializationApi::class)
+@OptIn(ExperimentalJsExport::class)
```

Comment:
Can we add this to the other Zipline subprojects and avoid annotations everywhere? Seems to work well for the CLI.

https://github.com/cashapp/zipline/blob/7a92b675ff53a1daa0bb3e8dcd474214fbe69ac9/zipline-cli/build.gradle.kts#L29-L35

Reply:
Since this thing’s a public API I wanna be more precise in where we apply these. Doing a global `ExperimentalSerializationApi` makes it easy to accidentally introduce an experimental API usage.